CHANGES.txt

```diff
@@ -40,15 +42,34 @@ Unreleased Changes
   * Fix FieldMaskUtil.subtract to recursively remove mask.
   * Mark enums with `@java.lang.Deprecated` if the proto enum has option
     `deprecated = true;`.
+  * Adding forgotten duration.proto to the lite library (#7738)
 
   Python
   * Print google.protobuf.NullValue as null instead of "NULL_VALUE" when it is
     used outside WKT Value/Struct.
   * Fix bug occurring when attempting to deep copy an enum type in python 3.
+  * Add a setuptools extension for generating Python protobufs (#7783)
+  * Remove uses of pkg_resources in non-namespace packages. (#7902)
+  * [bazel/py] Omit google/__init__.py from the Protobuf runtime. (#7908)
+  * Removed the unnecessary setuptools package dependency for Python package (#7511)
+
+  PHP
+  * Added support for "==" to the PHP C extension (#7883)
+  * PHP: Added `==` operators for Map and Array. (#7900)
```

Comment:
Maybe remove the `PHP:` and `[PHP]` prefixes as these are already in the PHP section?

Reply:
Done